Documentation/ceph-advanced-configuration.md

```diff
@@ -354,7 +354,10 @@ The default override settings are blank. Cutting out the extraneous properties,
 we would see the following defaults after creating a cluster:
 
 ```console
-$ kubectl -n rook-ceph get ConfigMap rook-config-override -o yaml
+kubectl -n rook-ceph get ConfigMap rook-config-override -o yaml
+```
+
+```console
```

Comment:
I think this is just text at this point, not console. We don't want to do any syntax highlighting.